backend/api/Startup.cs

```diff
@@ -124,6 +125,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSingleton<IAuthorizationHandler, RealmAccessRoleHandler>();
             services.AddTransient<IClaimsTransformation, KeyCloakClaimTransformer>();
+            services.AddTransient(s => s.GetService<HttpContext>().User);
```

Comment:
Not 100% sure this will work all the time.  I know there are issues with some DI related to the current context.  It has something to do with threading.  This may work, but my spider sense is saying I need to read up on it to make sure.